package.json

```diff
@@ -11,14 +11,15 @@
     "dev": "cross-env START_HOT=1 npm run start-renderer-dev",
     "flow": "flow",
     "flow-typed": "rimraf flow-typed/npm && flow-typed install --overwrite || true",
-    "lint-base": "eslint --cache --format=node_modules/eslint-formatter-pretty",
```

Comment:
why have you disabled caching? I found this to make linting quite slow?

Reply:
Thanks that was unintentional due to needing to disable caching when changing rules (because caching limits the run to just changed files, when the new lints should be run against all files when the rules change).